build/dependencies.props

```diff
@@ -79,7 +79,7 @@
     <HealthCheckRedis>2.2.3</HealthCheckRedis>
     <HealthCheckRabbitMQ>2.2.1</HealthCheckRabbitMQ>
     <HealthCheckEventStore>2.2.2</HealthCheckEventStore>
-    <HealthCheckElasticsearch>2.2.2</HealthCheckElasticsearch>
+    <HealthCheckElasticsearch>2.3.0</HealthCheckElasticsearch>
```

Comment:
Hi, when we need to upgrade version don't use "semantic versioning", for health checks we use 2.2.X because 2.2 is the version of .NET Core for this healthchecks (when ms release 3.0 we need to create package with 3.0.X), and X is our verion. If you address all the feedback please use 2.2.(++X)